src/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs

```diff
@@ -46,9 +50,18 @@ private unsafe bool StartWithShellExecuteEx(ProcessStartInfo startInfo)
             if (startInfo._environmentVariables != null)
                 throw new InvalidOperationException(SR.CantUseEnvVars);
 
+            if (startInfo != null && startInfo.ArgumentList.Count > 0)
+            {
+                arguments = PasteArguments.Paste(startInfo.ArgumentList, false);
```

Comment:
We generally like to include the name of the parameter for boolean parameters for better self-documentation: i.e.

  `arguments = PasteArguments.Paste(startInfo.ArgumentList, pasteFirstArgumentUsingArgV0Rules: false);`